src/operations/connect.ts

```diff
@@ -72,10 +72,7 @@ function createTopology(
   // save the reference to the topology on the client ASAP if the event handlers need to access it
   mongoClient.topology = topology;
 
-  // TODO(NODE-3273) - remove error
-  topology.once(Topology.OPEN, (error, topology) =>
-    mongoClient.emit(Topology.OPEN, error, topology)
-  );
+  topology.once(Topology.OPEN, () => mongoClient.emit('open', mongoClient));
```

Comment:
Here's the breaking change that is necessary to keep Topology `@internal` and not exposed in the type definitions. This is arguably a more useful API as the topology object that was emitted here doesn't have any of the useful, db/collections APIs most desire to use.